pkg/siteconfig/clusterinstance.go

```diff
@@ -11,10 +13,23 @@ import (
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	goclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var (
```

Comment:
I think I'd prefer the conditions being in the test suite. My reasoning is that conditions may change between releases (like cgu in 4.12)  so older ones are only available with older release branches of eco-goinfra. But cnf/ran always uses the latest version of eco-goinfra so we'd have to create our own variables for older conditions. If we're creating our own variables in the test suite then might as well only do that. Let me know if you disagree though I'm open to hearing alternatives

Reply:
Good point, I do agree the proposed approach to declare "ClusterInstanceConditionType" as variables in test suite instead of declaring it in the API/package(siteconfig) code.

Reply:
Taken care. changes are updated accordingly.